source/beamline/BioXAS/BioXASFilterFlipper.cpp

```diff
@@ -0,0 +1,256 @@
+#include "BioXASFilterFlipper.h"
+
+BioXASFilterFlipper::BioXASFilterFlipper(const QString &deviceName, QObject *parent) :
+	BioXASBeamlineComponent(deviceName, parent)
+{
+	// Create class variables.
+
+	slides_ = 0;
+	filters_ = 0;
+	slideChangeStatus_ = 0;
+	runMode_ = 0;
+	nextStepTrigger_ = 0;
+	receiverStatus_ = 0;
+	receiverStageCWLimit_ = 0;
+	receiverStageCCWLimit_ = 0;
+	cartridgeStageCWLimit_ = 0;
+	cartridgeStageCCWLimit_ = 0;
+}
+
+BioXASFilterFlipper::~BioXASFilterFlipper()
+{
+
+}
+
+bool BioXASFilterFlipper::isConnected() const
+{
```

Comment:
Thanks @iainworkman! I'll check these settings for both my laptop and the beamline machine.